docs/website/docs/dlt-ecosystem/destinations/snowflake.md

```diff
@@ -310,6 +310,28 @@ Above we set `csv` file without header, with **|** as a separator and we request
 You'll need those setting when [importing external files](../../general-usage/resource.md#import-external-files)
 :::
 
+### Query Tagging
+`dlt` tags sessions that execute loading jobs with following job properties:
+* **source** - name of the source (identical with the name of `dlt` schema)
+* **resource** - name of the resource
+* **table** - name of the table loaded by the job
+* **load_id** - load id of the job
+
+You can define query tag by defining a query tag placeholder in snowflake credentials:
```

Comment:
Is this a mandatory or an optional config? I.e. would the queries also get tagged in a default format without the config, or is the config required to enable query tagging in the first place?

Reply:
this is required to get any tagging. tagging requires roundtrip to the server and I do not want to enable it by default.
so: `query_tag` is None by default and tagging is disabled, you put your own string with placeholders to enable it

we probably add same feature to bigquery and redshift which support tagging as well